config/application.rb

```diff
@@ -15,6 +15,6 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
-    config.active_record.schema_format = :sql
```

Comment:
What is the advantage of using `schema.rb` over `structure.sql`? I'm not opposed to the change at all, just curious.